data/coalition/coalition news

```diff
@@ -0,0 +1,177 @@
+			"A news broadcast shows footage of many Heliarch agents arresting a group of terrorists. The Heliarch who was supervising the operation is later interviewed, explaining that the now prisoners were found to have stolen government equipment."
+
+news "heliarch propaganda"
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Heliarch propaganda"
+	message
+		word
+			"A short film plays on a monitor during a commercial break. A young Saryd walks in a forest, looking to his sides every few seconds to take in the view. Eventually he turns to see another Saryd, both surprised. As he approaches her and the two start talking, the camera pans out to a rising sun over the forest."
+			"A local screen shows a stressed Saryd working at a computer late at night. The door creaks, and she looks back to find her children peeking. She puts them to sleep, and then goes to bed herself. After waking up, she finds nobody around the house. She goes outside, seeing the children, along with their father, admiring the sun rising."
+			"After a government broadcast, a timelapse plays on a spaceport monitor, showcasing Ring of Friendship gaining more and more solar panels on its exterior over the centuries."
+
+news "longcow rancher"
+	location
+		attributes "arach" "longcows"
+	name
+		word
+			"Longcow rancher"
+	message
+		word
+			`"Caring for longcows, hard work it is. Too much space they take up. Too many legs."`
+			`"Never been around an angry longcow, have you? Word of advice, avoid their sides you should."`
+			`"Stopped using vaccines on longcows, long ago we did. Remember which leg to apply them on, we never could."`
+			`"Wish understand my jokes, my beauties could. Many kicks out of them, they would get."`
+			`"Fresh steak here! First leg steak, second leg steak... a discount for tenth leg steak there is!"`
+			"You see a rancher walk out of a shop with a trolley full of bells. You wonder if they are for multiple longcows, or just a particularly long one."
+
+news "kimek farmer"
+	location
+		attributes "kimek" "farming"
+	name
+		word
+			"Kimek farmer"
+	message
+		word
+			`"When starting out on this business, I was, lost in the farm I got. Tricky to tell where you are, it is, when green is all you see."`
+			`"A Heliarch agent, I once was. Too stressful, it was for me, so this job I took. Messages from colleagues, I still get, but if see the fruits I plant here, they could, definetly join me, they would."`
+			`"Some fresh produce would you like, Captain? If purchase these, you do, teach you a family recipe I could. I'm sure that love it, a human would. Probably."`
+			"After noticing you looking at the produce, the farmer asks if you would like some free samples, before grabbing an entire box with various fruits and vegetables, and handing it to you."
```

Comment:
```suggestion
			"After noticing you looking at the produce, the farmer asks if you would like some free samples, before grabbing an entire box with various fruits and vegetables and handing it to you."
```
That comma's _technically_ optional, but this is a spot where I'd recommend against it. Feels more smooth to me without it.